tests/queries/0_stateless/02267_type_inference_for_insert_into_function_null.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO function null() SELECT 1;
+INSERT INTO function null() SELECT number FROM numbers(10);
+INSERT INTO function null() SELECT number, toString(number) FROM numbers(10);
```

Comment:
Thank you!
LGTM.

But let's also add a test for `null('auto')` just to check that it also works, for consistency.

Reply:
OK.